main.go

```diff
@@ -18,6 +20,101 @@ import (
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
+const (
+	diskPath             string = "./tmp/disk.img.gz"
+	diskPathDecompressed string = "./tmp/disk.img"
+	diskPathConverted    string = "./tmp/disk.qcow2"
+)
+
+func apply(vmName string) error {
+	log.Println("Applying VirtualMachineExport object...")
+
+	yaml := fmt.Sprintf(`
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+  name: %s
+spec:
+  source:
+    apiGroup: "kubevirt.io"
+    kind: VirtualMachine
+    name: %s
+`, vmName, vmName)
+	cmd := exec.Command("kubectl", "apply", "-f", "-")
```

Comment:
please use kubernetes go client instead of exec command

Reply:
Yes, I'll take care of it in the another PR.